modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -159,9 +159,14 @@ sda.enkf.multisite <- function(settings,
 # Model Specific Setup ----------------------------------------------------
 
   #--get model specific functions
-  my.write_restart <- paste0("PEcAn.", model, "::write_restart.", model)
-  my.read_restart <- paste0("PEcAn.", model, "::read_restart.", model)
-  my.split_inputs  <- paste0("PEcAn.", model, "::split_inputs.", model)
+  #switch to depreciated version for debugging purposes
+  do.call("library", list(paste0("PEcAn.", model)))
+  my.write_restart <- paste0("write_restart.", model)
+  my.read_restart <- paste0("read_restart.", model)
+  my.split_inputs  <- paste0("split_inputs.", model)
```

Comment:
This looks kind of like you're trying to re-invent S3 dispatch.  Might be worth thinking about just implementing these functions as generic functions.  See http://adv-r.had.co.nz/S3.html

Reply:
Good catch this is depreciated version of the code that I use when debugging, I will update to current version.